installers/unix/env/install.sh

```diff
@@ -106,6 +112,28 @@ echo "Installing the OpenQuake Engine. Please wait."
 /usr/bin/env pip3 install --disable-pip-version-check wheelhouse/*.whl > /dev/null
 mkdir -p $FDEST/share
 cp -R src/{README.md,LICENSE,demos,doc} $FDEST/share
+cat <<EOF >> $FDEST/share/uninstall.sh
+#!/bin/bash
+
+if [ "\$1" != "-f" ]; then
+    while ! (echo "\$CONFIRM" | grep -qE '^[nNyY]$'); do
+        PROMPT="Are you sure you want to uninstall OQ and remove $FDEST? [y/n]: "
+        read -e -p "\$PROMPT" CONFIRM
+    done
+else
+    CONFIRM='y'
+fi
+
+if [[ "\$CONFIRM" == 'Y' || "\$CONFIRM" == 'y' ]]; then
```

Comment:
any reason to use double ``[`` ?

Reply:
No particular reason, but it would be `[ a ] || [ b ]` without double `[ ]`.

Reply:
Google says:

> Good Practice:
> Whenever you're making a Bash script, you should always use [[ rather than [.

Reply:
why not ``-a`` option?

Reply:
Ok, bash has its own ``[[`` builtin.